test/integration/scheduler_perf/config/performance-config.yaml

```
@@ -750,22 +753,33 @@
     labels: [integration-test, performance, short]
     params:
       initNodes: 5
+      initPods: 1
       measurePods: 10
   - name: 500Nodes/1kPods
     labels: [performance, short]
     params:
       initNodes: 500
+      initPods: 10
       measurePods: 1000
   - name: 5kNodes/1kPods
     labels: [performance, short]
     params:
       initNodes: 5000
+      initPods: 100
       measurePods: 1000
   - name: 5kNodes/10kPods
```

```suggestion
  - name: 5kNodes/100Init/10kPods
```
Should we keep the test names consistent?

---

Done.